capnslog/example/hello_dolly.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"github.com/coreos/pkg/capnslog"
+	oldlog "log"
+	"os"
+)
+
+var log = capnslog.NewPackageLogger("github.com/coreos/pkg/capnslog/cmd", "main")
+var dlog = capnslog.NewPackageLogger("github.com/coreos/pkg/capnslog/cmd", "dolly")
+
+func main() {
+	rl := capnslog.MustRepoLogger("github.com/coreos/pkg/capnslog/cmd")
+	rl.SetGlobalLogLevel(capnslog.INFO)
+	capnslog.SetFormatter(capnslog.NewGlogFormatter(os.Stderr))
+
+	// We can parse the log level configs from the command line
+	if len(os.Args) > 1 {
+		cfg, err := rl.ParseLogLevelConfig(os.Args[1])
+		if err != nil {
+			log.Fatalln(err)
+		}
+		rl.SetLogLevel(cfg)
+		log.Infoln("Setting output to", os.Args[1])
+	}
+
+	// Send some messages at different levels to the different packages
+	dlog.Infoln("Hello Dolly")
```

Comment:
- Why add the trailing `ln` to the method names?
- Why not just simplify the things down to a single method per level that always takes a fmt string (`Info(fmt string, args string...)`) and not make the caller choose between calling `Infoln` vs `Infof`? The number of times I've chosen the wrong one and messed up my logs is embarrassing.